### ISSUE TITLE
(inactive) pareg

### ISSUE THREAD

**Issue by kpj** [Bioconductor/Contributions#2534]
Update the following URL to point to the GitHub repository of
the package you wish to submit to _Bioconductor_

- Repository: https://github.com/cbg-ethz/pareg

Confirm the following by editing each check box to '[x]'

- [x] I understand that by submitting my package to _Bioconductor_,
  the package source and all review commentary are visible to the
  general public.

- [x] I have read the _Bioconductor_ [Package Submission][2]
  instructions. My package is consistent with the _Bioconductor_
  [Package Guidelines][1].

- [x] I understand Bioconductor [Package Naming Policy][9] and acknowledge
  Bioconductor may retain use of package name.

- [x] I understand that a minimum requirement for package acceptance 
  is to pass R CMD check and R CMD BiocCheck with no ERROR or WARNINGS. 
  Passing these checks does not result in automatic acceptance. The 
  package will then undergo a formal review and recommendations for 
  acceptance regarding other Bioconductor standards will be addressed.

- [x] My package addresses statistical or bioinformatic issues related
  to the analysis and comprehension of high throughput genomic data.

- [x] I am committed to the long-term maintenance of my package. This
  includes monitoring the [support site][3] for issues that users may
  have, subscribing to the [bioc-devel][4] mailing list to stay aware
  of developments in the _Bioconductor_ community, responding promptly
  to requests for updates from the Core team in response to changes in
  _R_ or underlying software.
  
- [x] I am familiar with the [Bioconductor code of conduct][7] and 
  agree to abide by it.

I am familiar with the essential aspects of _Bioconductor_ software
management, including:

- [x] The 'devel' branch for new packages and features.
- [x] The stable 'release' branch, made available every six
      months, for bug fixes.
- [x] _Bioconductor_ version control using [Git][5]
  (optionally [via GitHub][6]).

For questions/help about the submission process, including questions about
the output of the automatic reports generated by the SPB (Single Package
Builder), please use the #package-submission channel of our Community Slack.
Follow the link on the home page of the [Bioconductor website][8] to sign up.

[1]: https://contributions.bioconductor.org/
[2]: https://bioconductor.org/developers/package-submission/
[3]: https://support.bioconductor.org
[4]: https://stat.ethz.ch/mailman/listinfo/bioc-devel
[5]: http://bioconductor.org/developers/how-to/git/
[6]: http://bioconductor.org/developers/how-to/git/sync-existing-repositories/
[7]: https://bioconductor.org/about/code-of-conduct/
[8]: https://bioconductor.org/
[9]: https://bioconductor.org/developers/package-submission/#naming

**Comment by vjcitn** [comment]
You have edited the initial comment to add information.  Please restore the original "submission" comment format.  Your question about biocViews concerns a detail of formatting, I think -- 
```
biocViews: x,
     y
```
should work, but
```
biocViews:
   x,
   y
```
may not.  @LiNk-NY

**Comment by kpj** [comment]
Thanks a lot for your reply!

I have removed my comment from the submission. Here is the original comment:
> Note: my old submission was rejected due to "The package DESCRIPTION must contain a biocViews field with one or more
valid biocViews terms." which I find confusing because my DESCRIPTION has a biocViews field with valid terms.
Running `BiocCheck::BiocCheck('new-package'=TRUE)` locally also shows no problems with DESCRIPTION.
When comparing my project to a few others I noticed that I am using the master branch while others use the main branch. But the official [docs](https://bioconductor.org/developers/how-to/git/new-package-workflow/) also use the master branch, so this shouldn't be an issue, right?

The format
```
biocViews:
   x,
   y
```
should most definitely work. I am using it in one of my other [bioc packages](https://github.com/cbg-ethz/dce/blob/26a2eb9cc5a505f9938df3867f816855aa1df5a5/DESCRIPTION#L25-L34) and it is also used in another recent [package submission](https://github.com/PavlidisLab/gemma.R/blob/997f8355f0dd050456362252e5b784e96d2ec7d4/DESCRIPTION#L55-L68).
